h2o-genmodel/src/main/java/hex/genmodel/algos/drf/DrfMojoModel.java

```diff
@@ -1,13 +1,15 @@
 package hex.genmodel.algos.drf;
 
 import hex.genmodel.GenModel;
+import hex.genmodel.algos.tree.SharedTreeGraph;
 import hex.genmodel.algos.tree.SharedTreeMojoModel;
+import hex.genmodel.algos.tree.TreeBackedMojoModel;
 
 
 /**
  * "Distributed Random Forest" MojoModel
  */
-public final class DrfMojoModel extends SharedTreeMojoModel {
+public final class DrfMojoModel extends SharedTreeMojoModel implements TreeBackedMojoModel {
```

Comment:
should be specified on SharedTreeMojoModel level

Reply:
As stated above: Makes perfect sense. Moved to abstract predecessor. Removed from all the inheriting classes, including DrfMojoModel.